install.py

```diff
@@ -124,6 +127,9 @@ def main():
     prefix = "the relative path to the installation directory"
     parser.add_argument('--prefix', help=prefix, default=localdir)
 
+    config_only = 'only configure the package, do not build or install'
```

Comment:
@gidden, a quick search did not teach me what exactly this means. this works for me, I'm just trying to understand it.